pkgs/applications/misc/electrum-grs/default.nix

```diff
@@ -0,0 +1,130 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, wrapQtAppsHook
+, python3
+, zbar
+, secp256k1
+, enableQt ? true
+}:
+
+let
+  version = "4.1.5";
+
+  libsecp256k1_name =
+    if stdenv.isLinux then "libsecp256k1.so.0"
```

Comment:
Shouldn't there be a symlink to libsecp256k1.so? If not why not create it instead of hacking here?

Reply:
Context: https://github.com/NixOS/nixpkgs/pull/64636/commits/36f9da4ec43b849f913faefa52db49900ce59d53